src/debug/devtool/index.js

```diff
@@ -0,0 +1,36 @@
+export function initDevTools() {
+  // rdtHook -> React Devtools hook
+  const rdtHook = (window).__REACT_DEVTOOLS_GLOBAL_HOOK__;
+  
+  // the developer has not installed react devtools
+  if(rdtHook == null) return;
```

Comment:
when a person runs app in Development env, should we add a log recommending them to use React DevTools?

Reply:
Yes we should add that